db/migrate/20240530180734_create_active_storage_variant_records.active_storage.rb

```diff
@@ -0,0 +1,27 @@
+# This migration comes from active_storage (originally 20191206030411)
+class CreateActiveStorageVariantRecords < ActiveRecord::Migration[6.0]
```

Comment:
You can remove this migration too for similar reasons. Since this is a template project and there is no database, (and active storage isn't being used) we don't need to run this migration since there's no existing data that we need to migrate.

Reply:
Thanks for clarifying about these migration files. I've removed them, reset the schema to version 0, and re-ran `bin/setup` in a codespace to reset the `development.sqlite3` and `test.sqlite3` DBs and added those empty database files to this PR. These DBs were also already per-generated on `main` on a commit from last year: https://github.com/appdev-projects/rails-7-template/commit/1f54f85dea05d1271d89688523e2ec96c5dd51c1